grpc/tests/python/MyGame/Sample/Color.py

```diff
@@ -0,0 +1,9 @@
+# automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
We can't duplicate all these files from what is under `tests/`.. it should just refer to the existing files.

Reply:
Should I just upload the `monster_grpc_fb.py` in the `grpc/tests/python` folder or include it with the `tests/MyGame` folder?

Reply:
yes, `monster_grpc_fb.py` should still remain under `grpc/tests/python` much like the other languages.